src/modules/SpaceCalculation.rb

```diff
@@ -1052,7 +1052,23 @@ def size_from_string(size_str)
     # @return [Array<Storage::Filesystem>]
     def target_filesystems
       filesystems = Y2Storage::Filesystems::BlkFilesystem.all(staging_devicegraph)
-      filesystems.select! { |fs| fs.mountpoint && fs.mountpoint.start_with?("/") }
+      filesystems.select! do |fs|
+        # Ignore the devices mounted before starting the installer (e.g. the
+        # installation repository mounted by linuxrc when installing from HDD or
+        # the user mounted devices like for remote logging). Such devices will
+        # not be saved in the final /etc/fstab therefore check that flag.
+        # Check this only in the initial installation (as the non-fstab values
+        # will be missing in "/mnt"), in installed system they will stay available
+        # at "/".
+        # TODO: use a better API when provided by the libstorage-ng wrapper
```

Comment:
As discussed in irc? Why not create that better API right away? What about adding, e.g., a `Mountable#persistent_mount_point?` method to yast-storage-ng? Of course, I'm sure you can come up with a better name. :wink:

Reply:
yast2-storage-ng is there to evolve and adapt to the needs of the other YaST modules, not to be worked around. :-)